apps/common/components/SummaryData.tsx

```diff
@@ -0,0 +1,30 @@
+
+import type {ReactElement} from 'react';
+
+type TItem = {
+    label: string,
+    content: string
+}
+
+type TTabsProps = {
+	items: TItem[],
+}
+
+function SummaryData({items}: TTabsProps): ReactElement {
+	return (
+		<div className={'align-center flex w-full flex-row flex-wrap justify-center gap-14'}>
+			{items.map(((item): ReactElement => (
```

Comment:
nit: I think readability would improve a bit with destructuring this
```suggestion
			{items.map((({content, label}): ReactElement => (
```